### ISSUE TITLE
Minimum log format expected by sentry/raven-python

### ISSUE THREAD

**Issue by luzfcb** [getsentry/raven-python#1209]
Hello,this is probably a documentation error.

What is the minimum log format expected by sentry/raven-python?

In the raven python documentation, there are several formats, and I could not figure out what parameters are required and at what position within the log.

for example, in the [logging documentation](https://docs.sentry.io/clients/python/integrations/logging/), this format is used in the configuration example

```
[%(asctime)s][%(levelname)s] %(name)s %(filename)s:%(funcName)s:%(lineno)d | %(message)s
```

but in [raven-python django logging docs](https://docs.sentry.io/clients/python/integrations/django/#integration-with-logging), use another

```
%(levelname)s %(asctime)s %(module)s %(process)d %(thread)d %(message)s
```

[raven-python pyramid](https://docs.sentry.io/clients/python/integrations/pyramid/)
```
%(asctime)s,%(msecs)03d %(levelname)-5.5s [%(name)s] %(message)s
```

In short, I want to know how I can configure the `formatters` used by python-raven, so that the `formatter` provides as much information as possible for python-raven/Sentry, and in the end, make it possible to use the Sentry reports in their entirety, without a missing informations.
